lib/http-context.js

```diff
@@ -433,19 +433,26 @@ function toJSON(input) {
   }
 }
 
-function toXML(input) {
+function toXML(input, options) {
   var xml;
+  var xmlDefaultOptions = { declaration : true };
```

Comment:
AFAICT, this is adding another feature, which is to allow the developer to specify whether XML declaration is included in the response. I can live with the fact that it's introduced together with other changes, but please add a unit-test to verify.